arch/arm/src/rp2040/rp2040_start.c

```diff
@@ -41,7 +41,7 @@
  * Pre-processor Definitions
  ****************************************************************************/
 
-#define IDLE_STACK ((uint32_t)&_ebss+CONFIG_IDLETHREAD_STACKSIZE)
+#define IDLE_STACK ((uint32_t)_ebss+CONFIG_IDLETHREAD_STACKSIZE)
```

Comment:
```suggestion
#define IDLE_STACK ((uint32_t)_ebss + CONFIG_IDLETHREAD_STACKSIZE)
```
maybe even better to
```suggestion
#define IDLE_STACK ((uintptr_t)_ebss + CONFIG_IDLETHREAD_STACKSIZE)
```
since `const uintptr_t g_idle_topstack = IDLE_STACK;`

Reply:
Done.